src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../common/dom/fire_event";
+import "./ha-svg-icon";
+import { mdiChevronDown } from "@mdi/js";
+import { classMap } from "lit-html/directives/class-map";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: Boolean, reflect: true }) expanded = false;
+
+  @property({ type: Boolean, reflect: true }) outlined = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div class="summary" @click=${this._toggleContainer}>
+        <div class="summary-content">
```

Comment:
I don't think this div is needed? We can move the styles to the parent?

Reply:
done